### ISSUE TITLE
Accessibility: Radio Button Screen Reader Issue

### ISSUE THREAD

**Issue by jn394** [microsoft-search/pnp-modern-search#3902]
**Version used**
4

**Describe the bug**
While having a screen reader on and using the keyboard to navigate. While navigating the "New Types" widget each radio button will read out "Radio Button 1 of 1". 

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Turn on screen reader.
2. Press "Tab" to navigate to Widget and press "Enter".
3. Once widget is open press "Down Arrow" to go into the options.
4. While pressing the "Down Arrow", each option will read out "Radio Button 1 of 1"

**Expected behavior**
While using the screen reader each option should have its unique number followed by the total number of options. e.i. "Radio Button 1 of 10". 

**Screenshots**
<img width="393" alt="image" src="https://github.com/user-attachments/assets/a8ad07f0-5e69-4591-8d6e-acd735258783">

**Desktop (please complete the following information):**
 - Chrome

**Comment by wobba** [comment]
Fixed in #4805 (commit(s) `6a626300`, `dc682888`), now merged into the `develop` branch (merge commit `578dbb3`). The fix will ship in the next release; until then it can be validated by building from `develop`. Closing as resolved — please reopen if the issue persists after testing on `develop`.
